git-link.el

```diff
@@ -356,7 +369,9 @@ Defaults to \"origin\"."
   (interactive (list (git-link--select-remote)))
   (let* ((remote-host (git-link--remote-host remote))
 	 (commit      (word-at-point))
-	 (handler     (cadr (assoc remote-host git-link-commit-remote-alist))))
+	 (handler     (cl-loop for (key . value) in git-link-commit-remote-alist
+                               when (string-match-p key remote-host)
+                               finally return (car value))))
```

Comment:
Will do more testing with this before my next update. It was my first time using `cl-loop`. Will have a look at `cl-find` too (never used it).